src/ce.js

```diff
@@ -40,8 +40,9 @@ var ce = {
             'tag_name': elem.tagName
         };
 
-        if (_.includes(['input', 'select', 'textarea'], elem.tagName.toLowerCase())) {
-            props['value'] = this._getFormFieldValue(elem);
+        var formFieldValue = this._getFormFieldValue(elem);
+        if (_.includes(['input', 'select', 'textarea'], elem.tagName.toLowerCase()) && this._includeProperty(elem, formFieldValue)) {
```

Comment:
can we put `this._includeProperty` directly into `this._getFormFieldValue` and have `this._getFormFieldValue` return `null` if we shouldn't include the property? i want to avoid the case where someone else calls the function in the future and doesn't know to check it with `_includeProperty`. also can we default `val` to `null` in `_getFormFieldValue` so we don't end up returning undefined?